### PR TITLE
Read Availability zone from nodeAffinity requirements

### PR DESCRIPTION
**Pull request by reasonerjt**

### CODE REVIEW

File: pkg/backup/item_backupper.go [hunk 1]
```diff
@@ -432,7 +437,14 @@ func (ib *itemBackupper) takePVSnapshot(obj runtime.Unstructured, log logrus.Fie
 		log.Infof("label %q is not present on PersistentVolume, checking deprecated label...", zoneLabel)
 		pvFailureDomainZone, labelFound = pv.Labels[zoneLabelDeprecated]
 		if !labelFound {
+			var k string
 			log.Infof("label %q is not present on PersistentVolume", zoneLabelDeprecated)
+			k, pvFailureDomainZone = zoneFromPVNodeAffinity(pv, awsEbsCsiZoneKey, azureCsiZoneKey, zoneLabel)
```
Comment (sseago) [inline pkg/backup/item_backupper.go:442]: In my current cluster using ebs volumes, the nodeAffinity uses "failure-domain.beta.kubernetes.io/zone" in its `required.nodeselectorterms.matchexpressions` field. While this particular volume will be caught via the zoneLabelDeprecated *label* check, for completeness we should probably add `zoneLabelDeprecated` to the `zoneFromPVNodeAffinity` args.

**Review by sseago — CHANGES_REQUESTED** [review]
Other than adding one more arg to zoneFromPVNodeAffinity this looks good.

**Comment by jenting** [comment]
Is this PR superseded by #4338?

**Comment by reasonerjt** [comment]
@jenting it is. I'm closing this PR

**Pull request closed without merging**